crates/wash-cli/src/up/mod.rs

```diff
@@ -607,15 +644,14 @@ async fn stop_wasmcloud(mut wasmcloud_child: Child) -> Result<()> {
     Ok(())
 }
 
-async fn is_wadm_running(nats_opts: &NatsOpts, lattice_prefix: &str) -> Result<bool> {
-    let client = nats_client_from_opts(
-        &nats_opts.nats_host,
-        &nats_opts.nats_port.to_string(),
-        None,
-        None,
-        nats_opts.nats_credsfile.clone(),
-    )
-    .await?;
+async fn is_wadm_running(
+    nats_host: &str,
+    nats_port: u16,
+    credsfile: Option<PathBuf>,
+    lattice_prefix: &str,
+) -> Result<bool> {
+    let client =
+        nats_client_from_opts(nats_host, &nats_port.to_string(), None, None, credsfile).await?;
```

Comment:
This feels like it could be moved to `wash-lib`, what do you think?

Reply:
Turns out we have `create_nats_client_from_opts` in wash-lib already. I'm going to delete `nats_client_from_opts`